internal/worker/changestream/worker.go

```diff
@@ -170,10 +171,17 @@ func (w *changeStreamWorker) GetWatchableDB(ctx context.Context, namespace strin
 			return nil, errors.Trace(err)
 		}
 
-		mux, err := w.cfg.NewWatchableDB(w.cfg.AgentTag, db, fileNotifyWatcher{
-			fileNotifier: w.cfg.FileNotifyWatcher,
-			fileName:     namespace,
-		}, w.cfg.Clock, w.cfg.Metrics.ForNamespace(namespace), w.cfg.Logger)
+		mux, err := w.cfg.NewWatchableDB(
+			w.cfg.AgentTag,
+			db,
+			fileNotifyWatcher{
+				fileNotifier: w.cfg.FileNotifyWatcher,
+				fileName:     namespace,
+			},
+			w.cfg.Clock,
+			w.cfg.Metrics.ForNamespace(namespace),
+			w.cfg.Logger.Child(database.ShortNamespace(namespace)),
```

Comment:
suggestion: use `<model-name>-<short-uuid>` instead as we did for the model/*.log files.